style.user.css

```diff
@@ -43,7 +43,7 @@
 
     /* used by the side bars (start page and task detail view) */
     --reactist-bg-aside: var(--background-primary) !important;
-    --product-library-navbar-idle-fill: var(--reactist-bg-aside)
+    --product-library-navbar-idle-fill: var(--reactist-bg-aside);
```

Comment:
Oh, I missed that since I use the new layout..
Said new layout actually works like this and not with your change (the stock styles override mine)
Could you instead replace it with `var(--app-topbar-background-color) !important;`?
From my own testing, this works on both the old and the new layout.